charts/linkerd-crds/templates/policy/http-local-ratelimit-policy.yaml

```
@@ -0,0 +1,146 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: httplocalratelimitpolicies.policy.linkerd.io
+  annotations:
+    {{ include "partials.annotations.created-by" . }}
+  labels:
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    linkerd.io/control-plane-ns: {{.Release.Namespace}}
+spec:
+  group: policy.linkerd.io
+  names:
+    kind: HTTPLocalRateLimitPolicy
```

Do we need to define listKind (we do this for other types)?

```suggestion
    kind: HTTPLocalRateLimitPolicy
    listKind: HTTPLocalRateLimitPolicyList
```

---

Not currently used, but good to have :+1: